src/GraphQL/Utilities/SchemaPrinter.cs

```diff
@@ -1,3 +1,5 @@
+#pragma warning disable CS1591 // Missing XML comment for publicly visible type or member
```

Comment:
Did not spend any time on the SchemaPrinter, as it is marked with a todo "this should be completely rewritten", and I know we discussed having it use the AST printer from the parser project

Reply:
Yes.